lib/cucumber/core/events.rb

```diff
@@ -0,0 +1,69 @@
+require 'cucumber/core/event'
```

Comment:
Thanks, it's all about the [yardocs](http://www.rubydoc.info/github/cucumber/cucumber-ruby-core/Cucumber/Core/Events)